mmv1/third_party/terraform/data_sources/data_source_google_compute_instance_group_manager.go

```diff
@@ -0,0 +1,121 @@
+package google
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceGoogleComputeInstanceGroupManager() *schema.Resource {
+
+	return &schema.Resource{
+		Read: dataSourceComputeInstanceGroupManagerRead,
+		Schema: map[string]*schema.Schema{
```

Comment:
Why not use the `datasourceSchemaFromResourceSchema` method here? How did you choose these fields to add?

Reply:
using resource schema now.